x-pack/plugins/cases/server/authorization/utils.test.ts

```diff
@@ -276,4 +278,14 @@ describe('utils', () => {
       `);
     });
   });
+
+  describe('getAuthorizedAndUnauthorizedSavedObjects', () => {
+    it('partitions authorized and unauthorized cases correctly', () => {
```

Comment:
May be we can also check for correct result in case there aren't any `authorized` or `unAuthorized` Saved objects?

Reply:
Good catch! Thanks!